integration-cli/docker_cli_diff_test.go

```diff
@@ -78,15 +75,13 @@ func (s *DockerSuite) TestDiffEnsureOnlyKmsgAndPtmx(c *check.C) {
 	}
 
 	for _, line := range strings.Split(out, "\n") {
-		if line != "" && !expected[line] {
-			c.Errorf("%q is shown in the diff but shouldn't", line)
-		}
+		c.Assert(line != "" && !expected[line], checker.Equals, false)
```

Comment:
Could probably be splitted in 2 asserts (one with `Equals` and one with `True`) to be more readable

```go
c.Assert(line, checker.Equals, "")
c.Assert(expected[line], checker.True)
```